.github/workflows/SBOM.yaml

```
@@ -0,0 +1,35 @@
+name: Generate SBOM
+
+on:
+  push:
+    
+
+jobs:
+  generate_sbom:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2  #get depo code
+      
+      # SBOM generator JSON
+      - name: Generate Python SBOM
+        uses: CycloneDX/gh-python-generate-sbom@v2
+        with:
+          input: ./requirements.txt
+          output: ./generated_bom.json
+          format: json  # output format  (json)
+      # SBOM generator XML
+      - name: Generate Python SBOM
+        uses: CycloneDX/gh-python-generate-sbom@v2
+        with:
+          input: ./requirements.txt
+          output: ./generated_bom.xml
+          format: json  # output format  (xml)
+          
+      - name: Upload SBOM artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: sbom
+          path: |
+            ./generated_bom.json
+            ./generated_bom.xml
```

Always leave an empty line at the end
```suggestion
            ./generated_bom.xml

```